devel/Makefile

```diff
@@ -561,9 +559,9 @@ TMP_PREFIX = /tmp/mod_pagespeed.$(USER).install
 internal_release_test :
 	cd $(GIT_SRC) && \
 	  BUILDTYPE=Release install/run_program_with_ext_caches.sh \
-	    $(GIT_RELEASE_BIN)/mod_pagespeed_test  $(TEST_ARG) && \
+	    $(GIT_BIN)/mod_pagespeed_test  $(TEST_ARG) && \
```

Comment:
hmm, does this actually work? I think nothing overrides BUILDTYPE within Make for this target (unlike in internal_release_build where it passes it to MAKE)